tutorials/phase_contrast_calculation.ipynb

```diff
@@ -0,0 +1,216 @@
+      "text/plain": [
+       "<Figure size 720x720 with 2 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "pc_obj.plot()\n",
+    "\n",
+    "print(\"Mean Velocity: \" + str(pc_obj.mean_velocity_global) + \" cm/sec\")\n",
```

Comment:
I'd print these to fewer decimal places as the latter ones are meaningless anyway e.g. `print(f"Mean Velocity: {pc_obj.mean_velocity_global:.1f} cm/sec")`